apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -61,68 +76,49 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
   defp utxo_pos_for(%{call_data: %{utxo_pos: utxo_pos_enc}} = _event),
     do: Utxo.Position.decode!(utxo_pos_enc)
 
+  @spec do_new(map(), list(map())) :: t()
   defp do_new(contract_status, fields) do
     fields = Keyword.put_new(fields, :is_active, parse_contract_exit_status(contract_status))
     struct!(__MODULE__, fields)
   end
 
+  @spec make_event_data(Event.module_t(), Utxo.Position.t(), t()) :: struct()
   def make_event_data(type, position, %__MODULE__{} = exit_info) do
-    struct(type, exit_info |> Map.from_struct() |> Map.put(:utxo_pos, Utxo.Position.encode(position)))
+    struct(
+      type,
+      exit_info |> Map.from_struct() |> Map.put(:utxo_pos, Utxo.Position.encode(position))
+    )
   end
 
   # NOTE: we have no migrations, so we handle data compatibility here (make_db_update/1 and from_db_kv/1), OMG-421
-  def make_db_update(
-        {position,
-         %__MODULE__{
-           amount: amount,
-           currency: currency,
-           owner: owner,
-           exit_id: exit_id,
-           exiting_txbytes: exiting_txbytes,
-           is_active: is_active,
-           eth_height: eth_height
-         }}
-      )
-      when is_integer(amount) and is_integer(eth_height) and
-             is_binary(currency) and is_binary(owner) and is_integer(exit_id) and is_binary(exiting_txbytes) and
-             is_boolean(is_active) do
+  @spec make_db_update({Utxo.Position.t(), t()}) :: Utxo.Position.db_t()
+  def make_db_update({position, event}) do
```

Comment:
```suggestion
  def make_db_update({position, exit_info}) do
```

Reply:
Both make sense to me but `exit_info` is clearer. Changed in other methods for consistency except using `exit_event` for `new/2`